contribs/gmf/src/controllers/abstractdesktop.js

```diff
@@ -129,6 +146,94 @@ gmf.AbstractDesktopController = function(config, $scope, $injector) {
       .getFeatureOverlay();
   this.drawFeatureLayer.setFeatures(ngeoFeatures);
 
+  var ngeoFeatureHelper = $injector.get('ngeoFeatureHelper');
+
+  /**
+   * @type {ol.layer.Vector}
+   * @export
+   */
+  this.editFeatureVectorLayer = new ol.layer.Vector({
+    source: new ol.source.Vector({
+      wrapX: false,
+      features: new ol.Collection()
+    }),
+    style: function(feature, resolution) {
```

Comment:
That's a good idea.  It needs the `ngeoFeatureHelper`, though so I'm not sure where to put it.  Or, we could put it in the `ngeoFeatureHelper`, as one of its methods (i.e. not a static method).

What do you think ?